docs/sources/project/make-a-contribution.md

```diff
@@ -2,28 +2,78 @@ page_title: Make a project contribution
 page_description: Basic workflow for Docker contributions
 page_keywords: contribute, pull request, review, workflow, white-belt, black-belt, squash, commit
 
+
+<!-- TODO (@thaJeztah) remove after docs/base is updated -->
+<style type="text/css">
+.tg    {border-collapse:collapse;border-spacing:0;margin-bottom:15px;}
+.tg td {background-color: #fff;padding:5px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;vertical-align:top;}
+.tg th {font-weight:bold;padding:5px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;text-align:left;}
+.tg .tg-e3zv{width:150px;}
+</style>
+
+<style>
+
+/* GitHub label styles */
+.gh-label {
+    display: inline-block;
+    padding: 3px 4px;
+    font-size: 11px;
+    font-weight: bold;
+    line-height: 1;
+    color: #fff;
+    border-radius: 2px;
+    box-shadow: inset 0 -1px 0 rgba(0,0,0,0.12);
+}
+
+.gh-label.black-belt  { background-color: #000000; color: #ffffff; }
+.gh-label.bug         { background-color: #fc2929; color: #ffffff; }
+.gh-label.improvement { background-color: #bfe5bf; color: #2a332a; }
+.gh-label.project-doc { background-color: #207de5; color: #ffffff; }
+.gh-label.white-belt  { background-color: #ffffff; color: #333333; }
```

Comment:
these are added to mimic the github labels on the page :D and will only be used for that page,
so probably not interesting to put them in the central docs/base